### ISSUE TITLE
Discount Code Entry Failed

### ISSUE THREAD

**Issue by braxton-butcher** [searchdiscovery/loews-dl-spec-ga4#10]

**Comment by braxton-butcher** [comment]
@dbokadiya 

Can you please provide where I can trigger this event firing? On the QA site it looks like all discount codes are accepted

I moved this back to in progress, please move back to "SDI Review" once the scenario is provided. Thanks!

**Comment by dbokadiya** [comment]
@braxton-butcher It will fire on Calender page selects on date.

**Comment by braxton-butcher** [comment]
@dbokadiya Is there an example of a discount code that will fail? I see where to enter the code but no matter what I type "coupon_success" fires.

**Comment by JH-SDI** [comment]
@dbokadiya @brendenconcannon 

Moving this one to "to do". As @braxton-butcher mentioned, all codes are successful, even if I type random characters:



![Image](https://github.com/searchdiscovery/loews-dl-spec-ga4/assets/121498633/8d03cfed-ba50-453b-ad90-17ef97b1ee21)

This event should firer when the user attempt to apply an innaccurate or invalid Discount Code

**Comment by dbokadiya** [comment]
@JH-SDI Development is Done, @alexsaveljev @brendenconcannon Can you please help team to test this event.

**Comment by JH-SDI** [comment]
@dbokadiya 

The event is not firing because all the codes entered(even if we enter random characters) are accepted and "coupon_success" fires. 

Moved back to "to do"

**Comment by dbokadiya** [comment]
@brendenconcannon Kindly help to test this event.

**Comment by brendenconcannon** [comment]
@JH-SDI - did we try testing these on the 3 locations set up to accept bookings?

**Comment by JH-SDI** [comment]
@brendenconcannon 

Correct, basically it accepts any "coupon code" provided so it never returns an error or failed event: 



![Image](https://github.com/searchdiscovery/loews-dl-spec-ga4/assets/121498633/43d5b349-9cf2-4aa9-ac63-12aed0ff40a7)
![Image](https://github.com/searchdiscovery/loews-dl-spec-ga4/assets/121498633/69f471d1-f9e2-4eb2-aeb3-58e348b0a3ee)

Marked as "To Do"

**Comment by brendenconcannon** [comment]
@braxton-butcher - Coupon codes for CPKSUT, LBSYN, DI1AAA show available in pricing view 12/1-12/15 for New York and Chicago.

**Comment by braxton-butcher** [comment]
@brendenconcannon Are there any coupon codes that will fail on the dev site? No matter what we type in the coupon code returns as successful

**Comment by brendenconcannon** [comment]
@braxton-butcher - If that is the case we would probably have to fix the functionality in the UAT site and I am not sure that it would be worth the hours to the Loews team to do so. 

@dbokadiya - Can you see if there is a quick solution to debug the coupon functionality in the UAT BE?
